### DISCUSSION TITLE
async await GC

### DISCUSSION THREAD

**Discussion by b4lie** [KyryloKuzyk/PrimeTween#78] [General]
I saw in the document that async await will result in GC allocation, so you recommend using sequences. But have you ever considered supporting unitask extensions

**Comment by KyryloKuzyk** [comment]
PrimeTween already can work together with UniTask: you can `await` tweens and sequences like async methods, so all UniTask features that work with async methods also work with PrimeTween's tweens and sequences with 0GC.

Do you have any specific use cases where PrimeTween can be closely integrated with UniTask?

**Comment by KyryloKuzyk** [comment]
Hey @b4lie, UniTask is now supported since version 1.4.10. Either with an explicit `ToUniTask` method or with an implicit cast to UniTask.
